README.md

```diff
@@ -9,7 +9,44 @@ running in a cluster, such as rolling upgrades.
 To very concisely describe Aurora, it is like a distributed monit or distributed supervisord that
 you can instruct to do things like _run 100 of these, somewhere, forever_.
 
-
+## Medallia Features
+
+* Jobs using Docker containerizer doesn't use thermos-executor:
+  - Removes the requirement of having Python in the Docker Image.
+  - Use Docker image default CMD if the process cmdline is empty.
+* Different Instances of a Job can have different Docker Parameters. 
+  - specify a list of `Instance` in the `instance_variables` in the Job config.
```

Comment:
This line is super confusing if you haven't seen the implementation or used the feature yet.